lib/qbwc/response/customer_add_rs.rb

```diff
@@ -23,6 +23,7 @@ def process(config = {})
         objects = records.map do |object|
           { customers: {
             email: object['Name'],
+            name: object['Name'],
```

Comment:
Does this mean we were checking for the name field but it was never being retuned?

Reply:
yeah, exactly, right here:

```ruby
    def id_for_object(object, object_type)
      return object['id'] if object_type.nil?

      key = object_type.pluralize
      if key == 'customers'
        object['name']
      elsif key == 'payments'
        object['id']
      elsif key == 'shipments'
        object['name']
      elsif key == 'vendors'
        object['name'] || object['id']
      elsif PLURAL_PRODUCT_OBJECT_TYPES.include?(key)
        object['product_id']
      else
        object['id']
      end
    end
```